addons/google_calendar/security/google_calendar_security.xml

```diff
@@ -2,8 +2,8 @@
     <data noupdate="1">
         <record id="google_calendar_not_own_token_rule" model="ir.rule">
             <field name="name">Google Calendar NOT own token access rule</field>
-            <field name="model_id" ref="model_google_calendar_credentials"/>
-            <field name="domain_force">[('user_ids', 'not in', user.id)]</field>
+            <field name="model_id" ref="model_res_users_settings"/>
+            <field name="domain_force">[('user_id', '!=', user.id)]</field>
```

Comment:
Outch! No. You can't give anyone access to other user config (including token).
In which case is it needed?